index.html

```diff
@@ -38,19 +41,226 @@ <h1 class="boldtext">
             <a href="#"><i class="fa-brands fa-twitter fa-xl iconspace"></i></a>
           </li>
           <li>
-            <a href="#"><i class="fa-brands fa-linkedin-in fa-xl iconspace"></i></a>
+            <a href="#"
+              ><i class="fa-brands fa-linkedin-in fa-xl iconspace"></i
+            ></a>
           </li>
           <li>
-            <a href="#"><i class="fab fa-medium fa-xl iconspace iconspace"></i></a>
+            <a href="#"
+              ><i class="fab fa-medium fa-xl iconspace iconspace"></i
+            ></a>
           </li>
           <li>
-            <a href="#"><i class="fa-brands fa-github fa-xl iconspace "></i></a>
+            <a href="#"><i class="fa-brands fa-github fa-xl iconspace"></i></a>
           </li>
           <li>
             <a href="#"><i class="fab fa-angellist fa-xl iconspace"></i></a>
           </li>
         </ul>
       </div>
     </section>
+
+    <!-- works section starts here -->
+    <section class="works-section-container">
+      <section class="card-container card-type1">
+        <div class="image-container img1"></div>
+        <div class="content-container">
+          <h1 class="project-title">Tonic</h1>
+          <div class="frame2">
+            <h2>CANOPY</h2>
+            <ul class="card-role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+
+          <div class="description">
+            <p>
+              A daily selection of privately<br />
+              personalized reads; no accounts or<br />
+              sign-ups required.
+            </p>
+          </div>
+
+          <div >
+            <ul class="languages">
+              <li class="word-style">html</li>
+              <li class="word-style">css</li>
+              <li class="word-style">javaScript</li>
+            </ul>
+          </div>
+        </div>
+        <div class="action">
+          <button type="button" class="action-button">See Project</button>
+        </div>
+      </section>
```

Comment:
This is about every project card you have:

- [ ] It's better if you don't use `section` tag for every project because it doesn't really reflect a section and this would mess with the semantics of the website, kindly change that. 😉
- [ ] You shouldn't use `h1` for the title of the project, because `h1` can only be used once in your webpage and it's only used in the main title of the webpage, kindly change it to another `hx` tag. 😁

Reply:
removed section tag